lib/inferno/terminology/loader.rb

```diff
@@ -199,7 +199,8 @@ def save_new_value_set_list
             USCoreTestKit::USCoreV311::USCoreTestSuite.metadata +
             USCoreTestKit::USCoreV400::USCoreTestSuite.metadata +
             USCoreTestKit::USCoreV501::USCoreTestSuite.metadata +
-            USCoreTestKit::USCoreV610::USCoreTestSuite.metadata
+            USCoreTestKit::USCoreV610::USCoreTestSuite.metadata +
+            USCoreTestKit::USCoreV700::USCoreTestSuite.metadata
```

Comment:
This belongs in the terminology update PR. It's what updates the file in my last comment on that PR.

Reply:
Okay I will remove this here. I am currently working on addressing that last comment on the terminology PR and once I finish that, I will fix this here. I also see that I need to fix some linting issues